flake.nix

```diff
@@ -74,6 +74,19 @@
               inherit cargoArtifacts;
             });
           };
+        hercules-ci.flake-update = {
+          enable = true;
+          updateBranch = "updated-flake-lock";
+          # Next two parameters should always be set explicitly
+          createPullRequest = true;
+          autoMergeMethod = null;
```

Comment:
@szg251 can you please check how to perform auto merges for when the CI reports success?